custom/conf/app.example.ini

```diff
@@ -524,7 +524,7 @@ INTERNAL_TOKEN=
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;
 ;; Enables OAuth2 provider
-ENABLE = true
```

Comment:
Please also change `docs/content/administration/config-cheat-sheet.en-us.md ` to add a new config option.